stl/inc/type_traits

```diff
@@ -366,6 +366,18 @@ struct is_enum : bool_constant<__is_enum(_Ty)> {}; // determine whether _Ty is a
 template <class _Ty>
 _INLINE_VAR constexpr bool is_enum_v = __is_enum(_Ty);
 
+#if _HAS_CXX23
+// STRUCT TEMPLATE is_scoped_enum
+template <class _Ty, bool = conjunction_v<is_enum<_Ty>, negation<is_convertible<_Ty, int>>>>
+inline constexpr bool is_scoped_enum_v = true;
+
+template <class _Ty>
+inline constexpr bool is_scoped_enum_v<_Ty, false> = false;

```

Comment:
Couldnt this just be 
```suggestion
template <class _Ty>
inline constexpr bool is_scoped_enum_v<_Ty> = conjunction_v<is_enum<_Ty>, negation<is_convertible<_Ty, int>>>;
```

Reply:
That would be perhaps the more obvious solution...